src/main/java/uk/nhs/hee/tis/trainee/forms/api/FormRPartAResource.java

```diff
@@ -82,7 +82,7 @@ public FormRPartAResource(FormRPartAService service, FormRPartAValidator validat
   @PostMapping("/formr-parta")
   public ResponseEntity<FormRPartADto> createFormRPartA(@RequestBody FormRPartADto dto)
       throws URISyntaxException, MethodArgumentNotValidException {
-    log.debug("REST request to save FormRPartA : {}", dto);
+    log.info("REST request to save FormRPartA : {}", dto);
```

Comment:
I've made these all 'info' level logging because that's what we generally use, and it makes digging through the historic logs a bit easier to find what happened to particular user's forms. 'Debug' would only help with debugging a particular issue if we deployed specifically with that logging level, i.e. it wouldn't help diagnostics for past events.

But I could be wrong 😁